### ISSUE TITLE
Can't receive calls / related to color of dot next to username?

### ISSUE THREAD

**Issue by Questioning1** [tigase/beagle-im#145]
**Describe the bug**

I can call one of my colleagues but not the other, using Beagle-IM. The colleague that I can call shows up with a green or red dot indicating his status. The colleague that I can't call shows up with a greyish or white dot. When I try to call him it says "Call declined". This is true for both audio and video calls.
The weird thing is is that this colleague, whom I can't call, *can* call me, on both audio and video.
We have checked to the best of our ability all the settings in Beagle-IM.
Can anyone help us? I suspect that the color of the dot is related to the error that I get. But we can't figure out how to get his dot to change nor how to get him to receive a call.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS Monterey
 - Browser Firefox
 - Version 115.0.2

**Additional context**

**Comment by hantu85** [comment]
There is no white dot, that is always grey but due to the background colour it may look like white. This dot shows if user is online, ie. if his client is connected and available.

Grey dot mean, that user is not connected. In this case, the VoIP call is possible to this user only if he has client that supports push notifications for VoIP calls and when his server also supports VoIP push notifications.

To sum this up. Beagle informs you that the call was declined, due to the fact that it was not possible to establish the call as it was impossible to notify recipient about incoming all as his client was offline and it was no way to notify it.

**Comment by Questioning1** [comment]
> There is no white dot, that is always grey but due to the background colour it may look like white. This dot shows if user is online, ie. if his client is connected and available.
> 
> Grey dot mean, that user is not connected. In this case, the VoIP call is possible to this user only if he has client that supports push notifications for VoIP calls and when his server also supports VoIP push notifications.
> 
> To sum this up. Beagle informs you that the call was declined, due to the fact that it was not possible to establish the call as it was impossible to notify recipient about incoming all as his client was offline and it was no way to notify it.


Thanks for the response! Well, this is very mysterious, because the two colleagues are using the same server and the same client, so that is not a variable. I can call one of them but not the other, still. And the one I can't call is still showing up as grey. 

Any other suggestions welcome!
